packages/discord.js/src/managers/GuildBanManager.js

```diff
@@ -199,6 +199,31 @@ class GuildBanManager extends CachedManager {
     await this.client.rest.delete(Routes.guildBan(this.guild.id, id), { reason });
     return this.client.users.resolve(user);
   }
+
+  /**
+   * Ban up to 200 users from a guild, and optionally delete previous messages sent by the banned users. Requires
+   * both the `BAN_MEMBERS` and `MANAGE_GUILD` permissions.
+   * @param {BanBulkOptions} options The options for bulk banning users
+   * @returns {Promise<BanBulkResult>} Returns an object with `bannedUsers` key containing the IDs of the banned users
+   * and the key `failedUsers` with the IDs that could not be banned or were already banned.
+   * @example
+   * // Bulk ban users
+   * guild.bans.bulk({ users: ['84484653687267328'], deleteMessageSeconds: 604_800 })
+   *   .catch(console.error);
+   */
+  async bulk(options) {
+    if (typeof options !== 'object') throw new DiscordjsTypeError(ErrorCodes.InvalidType, 'options', 'object', true);
+    if (!Object.hasOwn(options, 'users')) throw new Error('Object "options" has no "users" attribute');
+    const { users } = options;
+
+    if (Array.isArray(users) || users instanceof Collection) throw new Error('"users" is not an array');
```

Comment:
```suggestion
    if (!Array.isArray(users) && !(users instanceof Collection)) throw new Error('"users" is not an array or collection');
```
should fix this line.